src/Athena/Views/Home/Add_Classes.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@{
+    ViewData["Title"] = "Add Classes";
+}
+
+<p>Use this area to provide additional information.</p>
+
+<!--http://jordnkr.github.io/collapsible/-->
+<div id="first">
+    <h1 id="specific" class="btn" style="width:30em;background-color:navy">Degree Specific Courses</h1>
+    <div>
+        <p>First Section Content</p>
+    </div>
+    <br />
+    <br />
+    <h1 class="btn" style="width:30em;background-color:navy">All Courses</h1>
```

Comment:
Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.